cluster/gce/config-default.sh

```diff
@@ -359,6 +359,7 @@ if [[ -n "${LOGROTATE_MAX_SIZE:-}" ]]; then
 fi
 
 # Fluentd requirements
+FLUENTD_GCP_VERSION="${FLUENTD_GCP_VERSION:-0.2-1.5.28-1}"
```

Comment:
This needs to be set also in config-test.sh, so that the tests won't fail. Please also add it to PROVIDER_VARS below.

Reply:
Updated, will see how the tests run, thanks!